XamarinCommunityToolkit/Behaviors/Animations/AnimationHelper.shared.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Behaviors.Animations
+{
+    public enum EasingType
```

Comment:
Why do we need another enum for this? If I look at [`GetEasing`](https://github.com/xamarin/XamarinCommunityToolkit/pull/169/files#diff-4760c85c886c7cec30b53abe78659c4fR22) all it does is mapping it to the Forms version? Why not use that directly?

Reply:
Agreed, let's stick to what we already have and limit the introduction of duplicate types.

Reply:
The enum is to make it easier for the user because the **Easing** of Xamarin Forms is a class, not an Enum. 
So, if I use the Xamarin.Forms Easing class, it would be more XAML lines and also not sure how to do it in XAML. 

I took the inspiration from Xam.Animations by @jsuarezruiz.

Reply:
Talked about this with @Char0394 its either doing this, or you have to specify the easing like this in your XAML: `<behaviors:FadeAnimation Easing="{x:Static Easing.Linear}" Duration="100"/>` because `Easing` is a class with static properties. Both is not great. 

I think for the latter there might be some magic we can do for XAML, but let's keep it like this for now and revisit later